io.catenax.generic.digital_product_passport/2.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1142 @@
+
+:SingleEntity a samm:Entity ;
+   samm:preferredName "Single Entity"@en ;
+   samm:description "Entity for a single concentration for the maximum or actual concentration."@en ;
+   samm:properties ( [ samm:property :concentrationSingle; samm:payloadName "value" ] ) .
+
+:footprintValue a samm:Property ;
+   samm:preferredName "Footprint Value"@en ;
+   samm:description "The value of the environmental footprint or carbon footprint of the product. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Annex I:\n(l) the environmental footprint of the product, expressed as a quantification, in accordance with the applicable delegated act, of a product?s life cycle environmental impacts, whether in relation to one or more environmental impact categories or an aggregated set of impact categories;\n(m) the carbon footprint of the product.\n"@en ;
+   samm:characteristic :FootprintValueCharacteristic ;
+   samm:exampleValue "12.678"^^xsd:float .
+
+:footprintRulebook a samm:Property ;
+   samm:preferredName "Footprint Rulebook"@en ;
+   samm:description "The applied rulebook for the environmental footprint of the product."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "https://www.alink.pdf/" .
+
+:footprintLifecycle a samm:Property ;
+   samm:preferredName "Footprint Lifecycle"@en ;
+   samm:description "The lifecycle stage, to which the environmental footprint corresponds. These could be for example \"raw material acquisition and pre-processing\", \"main product production\", \"distribution\" or \"end of life and recycling\"."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "main product production" .
+
+:footprintUnit a samm:Property ;
+   samm:preferredName "Footprint Unit"@en ;
+   samm:description "The unit of measurement of the environmental impact category. For each impact category a specific unit is used. For climate change/carbon footprint \"kg CO2 eq\" will be used. If an aggrergation is used, utilize the normalization and weighting mehtods used in the referenced rulebook."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "kg CO2 eq" .
+
+:footprintType a samm:Property ;
+   samm:preferredName "Footprint Type"@en ;
+   samm:description "The type of the environmental footprint of the product. This could be one of the environmental impact categories. For the carbon footprint chose the \"Climate Change\" category from the environmental impact categories. This attribute is mentioned in the ESPR proposal from March 30th, 2022 Annex I:\n(l) the environmental footprint of the product, expressed as a quantification, in accordance with the applicable delegated act, of a product?s life cycle environmental impacts, whether in relation to one or more environmental impact categories or an aggregated set of impact categories.\n(m) the carbon footprint of the product."@en ;
```

Comment:
this sounds like, there are pre-defined categories. Why not using an enum for them?

Reply:
This could be other categories depending on the standard. So this is kept open.